.github/workflows/terraform.yaml

```diff
@@ -1,28 +1,28 @@
-name: Check the Terraform Module
+name: Check Terraform Module
 
 on:
   workflow_call:
 
 jobs:
-  build:
+  check-terraform:
     name: Terraform Checks and Plans
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout the repository to the runner
+      - name: Checkout Repository
         uses: actions/checkout@v4
 
-      - name: HashiCorp - Setup Terraform
+      - name: Setup Terraform
         uses: hashicorp/setup-terraform@v3
 
-      - name: Terraform Format
+      - name: Format Terraform Files
         id: fmt
         run: terraform fmt -recursive -check
         continue-on-error: true
```

Comment:
Why do we have this stage at all? This is used to reformat the terraform files but we won't do anything with the output

Reply:
`terraform fmt` output is processed.